en/gps_compass/septentrio_mosaic-go.md

```diff
@@ -0,0 +1,230 @@
+# Septentrio mosaic-go evaluation kit
+
+The mosaic-go Evaluation Kit is composed of the mosaic module soldered on an interface board inside a metallic housing.
+
+Sized at only 71 x 59 x 12 mm and weighing only 58 g, mosaic-go offers unmatched size to performance ratio. mosaic-go
+includes:
+
+- High update rate (>100 Hz) and low latency, both crucial for control systems of autonomous applications
+- Reliable centimetre-level positioning
+- Full L2 support via P(Y) code
+
+  Mosaic-go part number:
+  Single-antenna version, incorporating mosaic-X5: 410386(including accessories). Dual-antennaversion,
+  incorporatingmosaic-H: 410397(including accessories).
+
+- Size: 71 x 59 x 12 mm ± 1mm
+- Weight: 58g ± 1g
+
+<div style="text-align: center;">
+
+![Highly accurate GNSS receiver module](../../assets/hardware/gps/septentrio_sbf/mosaic-go.png "Highly accurate GNSS receiver module")
+
+</div>
```

Comment:
Please remove **all** alignment div's. The build toolchain provides consistent look and feel for all docs.
 
```suggestion
![Highly accurate GNSS receiver module](../../assets/hardware/gps/septentrio_sbf/mosaic-go.png "Highly accurate GNSS receiver module")
```